src/WinAppDriver/ClickElementHandler.cs

```diff
@@ -5,23 +5,14 @@ namespace WinAppDriver
     using System.Runtime.InteropServices;
     using System.Windows;
     using System.Windows.Automation;
+    using WinUserWrapper;
 
     [Route("POST", "/session/:sessionId/element/:id/click")]
     internal class ClickElementHandler : IHandler
     {
-        private const int MouseEventFlagMove = 0x0001;
-        private const int MouseEventFlagLeftdown = 0x0002;
-        private const int MouseEventFlagLeftup = 0x0004;
-        private const int MouseEventFlagRightdown = 0x0008;
-        private const int MouseEventFlagRightup = 0x0010;
-        private const int MouseEventFlagMiddledown = 0x0020;
-        private const int MouseEventFlagMiddleup = 0x0040;
-        private const int MouseEventFlagAbsolute = 0x8000;
-
         private static ILogger logger = Logger.GetLogger("WinAppDriver");
-        
-        [DllImport("user32.dll")]
-        public static extern void SetCursorPos(int x, int y);
+
+        private IWinUserWrap winUser = new WinUserWrap();

```

Comment:
The dependency should be injected instead. How about creating a IMouse as a wrapper around underlying Win32 APIs?